integration/shareddata/scalars.go

```diff
@@ -312,10 +292,10 @@ var Int32s = &Values[string]{
 		"int32-zero": int32(0),
 		"int32-max":  int32(math.MaxInt32),
 		"int32-min":  int32(math.MinInt32),
-		"int32-null": nil,
-		"int32-1":    int32(4080),
-		"int32-2":    int32(1048560),
-		"int32-3":    int32(268435440),
+		// "int32-null": nil, TODO: uncomment when Tigris fixes this
```

Comment:
Should we add a link to any issue here?

Reply:
There is a chance Tigris can fix it quickly, so if they release a new beta soon (e.g. today), I'd just update it in this PR :)

Reply:
Added issue number